docs/source/repositories.md

```diff
@@ -0,0 +1,48 @@
+# Repositories Overview
+
+The **Leapp** project uses a modular repository structure in `repos/system_upgrade` to support multiple upgrade paths
```

Comment:
```suggestion
This project uses a modular repository structure in `repos/system_upgrade` to support multiple upgrade paths
```

Well, I would not speak here about how the leapp framework works. The leapp project is reference actually to the leapp git repository and so to the framework or tooling. In this case, we could speak more e.g. about the **upgrade** project unless we add in future more stuff unrelated to the upgrade. So far, there is no such expectation so saying `the upgrade project` would work too.